README.md

```diff
@@ -1,11 +1,11 @@
-# Mobile Test Runner For Jama
+# Mobile Test Runner For Jama Software
 
-The Mobile test runner for Jama app is an iPhone application that will allow the users of the Jama product development platform to easily execute test plans, and test cases to manage testing in real-time. Users can view run, and submit their test cases.
+The Mobile test runner for Jama Software app is an iPhone application that will allow the users of the Jama Software product development platform to easily execute test plans, and test cases to manage testing in real-time. Users can view run, and submit their test cases.
```

Comment:
Can we change the wording here? We reference executing test cases which is not very accurate and since Test Cases refer to an object in Jama that we are not dealing with I think it could be confusing.
Maybe something like:
```The Mobile test runner for Jama Software app is an iPhone application that will allow the users of the Jama Software product development platform to easily execute and manage their assigned test runs in real-time. Users can view, run, and submit their assigned test runs.```